packages/core/components/AggregateInfoBox/index.tsx

```diff
@@ -85,10 +86,7 @@ export default function AggregateInfoBox() {
                         {!isLoading && aggregateData ? (
                             aggregateData.count
                         ) : (
-                            <Spinner
-                                size={SpinnerSize.small}
-                                data-testid="aggregate-info-box-spinner"
-                            />
+                            <LoadingIcon testId="aggregate-info-box-spinner" />
```

Comment:
Added a common component because we were doing some complicated styling in many many places and it was funky and inconsistent